src/material-experimental/mdc-tabs/testing/tab-nav-panel-harness.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {TabNavPanelHarnessFilters} from './tab-harness-filters';
+
+/** Harness for interacting with a standard mat-tab-nav-panel in tests. */
+export class MatTabNavPanelHarness extends ComponentHarness {
```

Comment:
For components like this one that have projected content, we usually extend the `ContentContainerComponentHarness`.

Reply:
Done.